### PR TITLE
fix: make bundleHash from relay response optional

### PR DESCRIPTION
**Pull request by 0xdapper**

### CODE REVIEW

**Comment by epheph** [comment]
When is it not returned? When using a non-flashbots relay?

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "build": "npx tsc",
     "clean": "rm -rf build/",
     "prepare": "npm run clean && npm run build",
-    "lint": "npx eslint src"
+    "lint": "npx eslint src",
+    "postinstall": "npm r build"
```
Comment (epheph) [inline package.json:17]: npm remove build ? what's this about?

Comment (0xdapper) [reply package.json:17]: should've been `run`, but will remove this entirely

**Comment by pawurb** [comment]
Is there any chance to merge it? I've confirmed that some builders (eg. rsync-builder),  return `null` response, even if submitted bundles end up on chain.

**Comment by manishparanjape** [comment]
Any update on this? I also have confirmed that Flashbots Builder returns null as well

**Pull request merged**